src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -190,6 +195,38 @@ def get_inventory_script(inventory):
     return inventory
 
 
+def get_callback_plugin_dir(test_env):
+    """Determine callback plugins path, create if missing, set env. var."""
+    callback_plugin_dir = os.path.join(
+        os.environ["TOX_WORK_DIR"], "callback_plugins"
+    )
+    if not os.path.isdir(callback_plugin_dir):
+        os.makedirs(callback_plugin_dir)
```

Comment:
`exists_ok=True` simplifies this and avoids TOCTOU. https://docs.python.org/3/library/os.html#os.makedirs

Reply:
`Changed in version 3.2: Added the exist_ok parameter.`

Not sure if we can drop support for python 2 yet.

Reply:
This only runs on testing hosts, which at the moment is only GitHub's `ubuntu-latest` (doesn't even have python 2) and our devloper machines (hopefully don't have Python 2 any more either). So I thought "latest py3 features" would be ok for this?

Reply:
> This only runs on testing hosts, which at the moment is only GitHub's `ubuntu-latest` (doesn't even have python 2) and our devloper machines (hopefully don't have Python 2 any more either). So I thought "latest py3 features" would be ok for this?

ok - I guess we can try and see what happens - note that there are many places in the code which does `isdir` first - should I change all of them?